src/avram/query_builder.cr

```diff
@@ -74,7 +74,7 @@ class Avram::QueryBuilder
       else
         value.to_s
       end
-    end
+    end.to_a
```

Comment:
This is called at the end of `.map` which should return an array already. @bcardiff does this help to ensure the type is always an array?

Reply:
Yes. Since the `args:` named keyword is restricted to arrays since from a user perspective if it was a tuple the arguments could be splatted directly.

Reply:
oh, that makes sense.

Reply:
I might be overlooking some designs of avram though. From what I understood all the query building goes through the same process. Despite knowing by construction that the query will have only one argument: `User.find(1)`. I think that pushed the dynamic aspect of the arguments to all the queries. Maybe things can be done differently.